src/main/kotlin/com/jetbrains/snakecharm/inspections/SmkMultilineFunctionCallInspection.kt

```diff
@@ -0,0 +1,76 @@
+package com.jetbrains.snakecharm.inspections
+
+import com.intellij.codeInspection.*
+import com.intellij.lang.ASTNode
+import com.intellij.openapi.command.WriteCommandAction
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiElement
+import com.intellij.psi.TokenType
+import com.intellij.psi.util.elementType
+import com.jetbrains.python.PyElementTypes
+import com.jetbrains.python.PyTokenTypes
+import com.jetbrains.python.psi.PyArgumentList
+import com.jetbrains.snakecharm.SnakemakeBundle
+import com.jetbrains.snakecharm.lang.psi.SmkRuleOrCheckpointArgsSection
+
+class SmkMultilineFunctionCallInspection : SnakemakeInspection() {
+    override fun buildVisitor(
+        holder: ProblemsHolder,
+        isOnTheFly: Boolean,
+        session: LocalInspectionToolSession
+    ) = object : SnakemakeInspectionVisitor(holder, session) {
+        override fun visitSmkRuleOrCheckpointArgsSection(st: SmkRuleOrCheckpointArgsSection) {
+            val args = st.children.firstOrNull { it is PyArgumentList } ?: return
+
+            if (multilineSectionDefinition(args as PyArgumentList)) {
+                return
+            }
+
+            val problemCalls = arrayListOf<PsiElement>()
```

Comment:
here whitespaces are collected, not PyCallExpressions, so better to rename variable into `invalidWhitespaces`